components/input/src/index.js

```diff
@@ -32,8 +32,8 @@ const GInput = glamorous.input(
       outlineOffset: 0,
     },
   },
-  ({ error }) => ({
-    border: error ? `4px solid ${ERROR_COLOUR}` : undefined,
+  ({ error, errorColour }) => ({
```

Comment:
`Colour` here kinda grates with me a bit - as this is styling i'd personally prefer `Color`
maybe we should support both, and `||` them?

Reply:
I agree, I was being consistent with what we've used previously, but I'd be happy to standardize on color